### ISSUE TITLE
subregion1_name errors in England

### ISSUE THREAD

**Issue by mpbrenner** [GoogleCloudPlatform/covid-19-open-data#238]
in the index.csv file the subregion1_names in england have errors:

--'Yorkshire and the Humber, England', instead of 'Yorkshire and the Humber':
-- 'East Midlands, England', instead of  'East Midlands'
--   'West Midlands, England' instead of 'West Midlands'

**Comment by themonk911** [comment]
@owahltinez I think this was an intentional change in your recent commit?

**Comment by owahltinez** [comment]
Yes, that's correct. @mpbrenner are you dependent on the names of these locations? Is there any way for you to use the key instead? We got some feedback about our subregions in England being confusing, so we added the ", England" to some of them for clarification (since they are children locations from GB, not GB_ENG)

**Comment by owahltinez** [comment]
Closing since it's working as intended
